requirements.txt

```diff
@@ -8,3 +8,4 @@ socketIO-client == 0.5.6
 stopit >= 1.1.1, < 1.1.999
 yamlconf >= 0.0.3, < 0.0.999
 revscoring >= 0.7.10
+redis
```

Comment:
I don't see where the test config wants it.  Were you unable to run the test server without it?